### PR TITLE
Add some new translations and fix the wrong one

### PR DESCRIPTION
**Pull request by TrueRou**

### CODE REVIEW

**Comment by RiskyKen** [comment]
Thanks. If you have a curse forge account let me know and I will add you to the contributors list. https://minecraft.curseforge.com/projects/dakimakura-mod

File: src/main/resources/assets/dakimakuramod/lang/zh_CN.lang [hunk 1]
```diff
@@ -1,77 +1,83 @@
...
-config.dakimakuramod:mobDropLootingBonus.tooltip=设置抱枕设计图在每级抢夺附魔下增加的掉落几率, 实际掉落几率 = 基础掉落几率 + 抢夺增加几率 * 抢夺等级
-
-config.dakimakuramod:textureMaxSize=最大贴图尺寸.
-config.dakimakuramod:textureMaxSize.tooltip=抱枕贴图尺寸.\n这个值将被四舍五入到最合适比例 2.\n将限制在GPU最大纹理尺寸.
-
-config.dakimakuramod:dakiRenderDist=最大渲染距离.
-config.dakimakuramod:dakiRenderDist.tooltip=显示抱枕的最大距离(单位为方块).
-
-config.dakimakuramod:checkForUpdates=是否自动检查更新
-config.dakimakuramod:checkForUpdates.tooltip=打开Minecraft时是否自动检测Mod有新版本
```
Comment (RiskyKen) [inline src/main/resources/assets/dakimakuramod/lang/zh_CN.lang:77]: Can you re-add this line? It's still used in the 1.7.10 version.

**Review by RiskyKen — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/dakimakuramod/lang/zh_CN.lang (hunk 1 above)
Comment (TrueRou) [reply src/main/resources/assets/dakimakuramod/lang/zh_CN.lang:77]: Sorry, I re-added it

**Comment by RiskyKen** [comment]
Thanks new build should be out later today.

**Pull request merged**